composer.json

```diff
@@ -14,14 +14,14 @@
     },
     "require": {
         "php": ">=5.5",
-        "ext-ldap": "*",
-        "zendframework/zend-stdlib": "~2.5"
+        "ext-ldap": "*"
     },
     "require-dev": {
         "zendframework/zend-config": "~2.5",
         "zendframework/zend-eventmanager": "~2.5",
         "fabpot/php-cs-fixer": "1.7.*",
-        "phpunit/PHPUnit": "~4.0"
+        "phpunit/PHPUnit": "~5.0",
```

Comment:
you can use `"phpunit/phpunit": "~4.0|~5.0"` then the dependency of php >=5.6 will use phpunit 5, and for php 5.5 will still uses phpunit 4